lib/worker.js

```diff
@@ -0,0 +1,68 @@
+const {Task} = require('atom');
+const {default: PQueue} = require('p-queue');
+const uniqueString = require('unique-string');
+
+const queue = new PQueue({concurrency: 1});
+let worker = null;
```

Comment:
```suggestion
let worker;
```

I prefer to not use `null`.